mlkem/native/x86_64/shuffle.inc

```diff
@@ -13,12 +13,19 @@ vpunpcklqdq	%ymm\r1,%ymm\r0,%ymm\r2
 vpunpckhqdq	%ymm\r1,%ymm\r0,%ymm\r3
 .endm
 
+// Shuffle r0=(a0,b0,c0,d0,...), r1=(a1,b1,c1,d1,...) into
+// r2 = (a0,b0,a1,b1,e0,f0,e1,f1,...)
+// r3 = (c0,d0,c1,d1,g0,h0,g1,h1,...)
```

Comment:
Do you want to stick with `/*` for C90-compliance - the assembly files do not seem to go throught he compiler, but given that you have used `/* */` throughout the rest of the code, you may want to change it.

Reply:
Changed this